### DISCUSSION TITLE
Set pull request reviewers on weblate config/level

### DISCUSSION THREAD

**Discussion by quintanilha6** [WeblateOrg/weblate#10325] [Ideas]
Would be nice if we could somehow add Pull Request reviewers on Weblate level. For example, some of the users I have set on my Weblate component should review the changes on the Pull request level (on Bitbucket for example). When there is a PR from Weblate, no reviewers are added automatically. 

I'm aware I could set "Default reviewers" on my Bitbucket repository, but that is not exactly what we are looking for.

**Comment by nijel** [comment]
I don't know how it works on Bitbucket, but on GitHub, Weblate would need repository write permissions to be able to assign a reviewer on a pull request. Presently, it doesn't need to ask for any repository-specific permissions on public repos. It's not a blocker, but it is definitely something which needs to be evaluated.

On GitHub this can be addressed by CODEOWNERS file in the repository, see https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

**Reply by quintanilha6** [reply]
On our bitbucket repo we provide writing permissions to Weblate. Now what I would like to do is, somewhere in Weblate's interface set up who should be added as a reviewer when I create a pull request from Weblate.

**Reply by nijel** [reply]
There already is https://github.com/WeblateOrg/weblate/issues/8494 covering this kind of setup.
